src/uct/base/uct_md.h

```diff
@@ -105,8 +105,8 @@ typedef ucs_status_t (*uct_md_mem_query_func_t)(uct_md_h md,
                                                 size_t length,
                                                 uct_md_mem_attr_t *mem_attr);
 
-typedef ucs_status_t (*uct_md_mkey_pack_func_t)(uct_md_h md, uct_mem_h memh,
-                                                void *rkey_buffer);
+typedef ucs_status_t (*uct_md_mkey_pack_func_t)(uct_md_h md,
```

Comment:
should not we define a new func, like `pack_func_v2`?

Reply:
no need because v2 is only on api level. this function definition is internal.